src/App.css

```diff
@@ -57,6 +57,7 @@
 
 .attribution-text{
```

Comment:
A minor point which I should have made as part of the attribution PR -- if the attribution-specific CSS were moved to a separate `attribution.css` file, it would be that much easier to reuse it in another context. No need to do anything about it now, but wanted to get it out there.